docs/Guide.RunningOnCI.md

```diff
@@ -80,3 +82,95 @@ script:
 - detox test --configuration ios.sim.release --cleanup
 
 ```
+
+### • Running Detox on [Bitrise](https://www.bitrise.io/)
+
+Bitrise is a popular CI service for automating React Native apps. If you are looking to get started with Bitrise, check out [this](http://blog.bitrise.io/2017/07/25/how-to-set-up-a-react-native-app-on-bitrise.html) guide.
+
+You can run Detox on Bitrise by adding a `tests` workflow. Here's what the Bitrise **.yml** file looks like for doing so:
```

Comment:
I would like to remove the `tests` here, the name doesn't matter and it might lead to confusion

Reply:
true. I'll remove `tests` and add a more general explanation instead.